proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol

```diff
@@ -115,6 +115,34 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
      */
     event TransferSkipped(uint256 id);
 
+    /**
+     * @dev Emitted when big transfer threshold is changed
+     */
+    event BigTransferThresholdIsChanged(
+        bytes32 indexed schainHash,
+        address token,
```

Comment:
Can you make this parameter indexed too?

Reply:
OK.